contrib/slurm/slurmexecutor.py

```diff
@@ -40,9 +40,10 @@ def get_system_info():
 
 
 def execute_benchmark(benchmark, output_handler):
-    num_of_cores = benchmark.rlimits.cpu_cores
-    mem_limit = benchmark.rlimits.memory
-    run_sets_executed = 0
+
+    assert (
+        not benchmark.config.use_hyperthreading
+    ), "SLURM can only work properly without hyperthreading enabled. See README.md for details."
```

Comment:
An `assert` will trigger a crash with an ugly stack trace shown to the user. Can you please use `sys.exit`? And mention that they need to pass `--no-hyperthreading`, like `Pass --no-hyperthreading and see README.md for details.`.